kalite/i18n/management/commands/makemessages.py

```diff
@@ -30,32 +33,42 @@ def handle_noargs(self, *args, **options):
         for ignore_pattern in IGNORE_PATTERNS:
             if ignore_pattern not in options["ignore_patterns"]:
                 options["ignore_patterns"].append(ignore_pattern)
-        print("Ignoring the following patterns: {}".format(options["ignore_patterns"]))
+        logging.debug("Ignoring the following patterns: {}".format(options["ignore_patterns"]))
 
-        print("Calling base makemessages command.")
+        logging.debug("Calling base makemessages command.")
         super(Command, self).handle_noargs(*args, **options)
+        lang_code = options["locale"]
+        logging.debug("Modifying message catalog for locale '{}'".format(lang_code))
+
+        # only add the inline help narratives for the JS catalog file.
+        if options["domain"] == "djangojs":
+            inline_help_poentries = self.extract_inline_help_strings()
+
+            pofile_path = os.path.join(settings.USER_WRITABLE_LOCALE_DIR, lang_code, "LC_MESSAGES", "djangojs.po")
+            po = polib.pofile(pofile_path)
+            for entry in inline_help_poentries:
+                po.append(entry)
+            po.save(pofile_path)
 
+    def extract_inline_help_strings(self, inline_help_path=None):
+        '''
+        Extract the strings from the inline help narratives yml files. Returns an
+        iterator containing the po file entries. Optional inline_help_parameter
+        specifies where the inline help narratives path is. Else, it defaults
+        to settings.CONTENT_DATA_PATH + "narratives.yml"
```

Comment:
>Optional inline_help_parameter
specifies where the inline help narratives path is. Else, it defaults
to settings.CONTENT_DATA_PATH + "narratives.yml"

It appears to be an unused parameter.